config/util/representation-conversion/default.json

```diff
@@ -15,18 +15,13 @@
       "@id": "urn:solid-server:default:RepresentationConverter",
       "@type": "WaterfallHandler",
       "handlers": [
-        { "@id": "urn:solid-server:default:MarkdownToHtmlConverter" },
         { "@id": "urn:solid-server:default:DynamicJsonToTemplateConverter" },
```

Comment:
Why still this one though? Maybe just get rid of the waterfall?

Reply:
The `DynamicJsonToTemplateConverter` is a special case because its output types depend on the metadata of the incoming Representation (it extracts templates from that) so it doesn't work as a `TypedRepresentationConverter` unfortunately. Currently this one is only used to generate the HTML templates in the IDP component.

Reply:
This comment made me realize that there actually is an issue with this converter now. If you request `application/json` from the IDP you're still going to get JSON, but if you request `application/json, text/html;q=0.5` you're going to get html. It's easy to fix it in that class but it is unfortunate that this is not part of the conversion chain.

Reply:
And `application/json` is obviously a lie, because the content type is more specific 🙂
But good to leave as-is.